@types/interfaces.d.ts

```diff
@@ -0,0 +1,13 @@
+export interface TimePickerTime {
+    hours: number;
```

Comment:
Please change indentation to use 2 spaces.

Reply:
Since the last update this interface has 2 space indentation but the i18n interface is still indented with 4 spaces.

Reply:
Done